packages/Adapters/POD_C/src/DTK_C_API.h

```diff
@@ -0,0 +1,49 @@
+#ifndef DTK_NONAME_H
+#define DTK_NONAME_H
+
+#include "mpi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// QUESTION: Do we want to be able to track missed entities?
+
+// Convenience typedef for an opaque pointer
+typedef void DTK_Map;
```

Comment:
I think we can get away with just this typedef. I haven't seen a need for an additional struct yet.